src/test/java/com/google/javascript/cl2dts/provide_single_class.js

```diff
@@ -6,6 +6,11 @@ goog.provide('foo.bar.Baz.NestedEnum');
 foo.bar.Baz = function() {
   /** @type {string} */
   this.field = 'a';
+  // Surprisingly, this doesn't throw a type error.
```

Comment:
maybe not just `this`, but `declaring a field on the prototype and in the constructor` to make it a bit clearer?